mrmap/service/templatecodes.py

```diff
@@ -27,6 +27,13 @@
       {{ ICONS.DELETE|safe }}
     </a>
     {% endif %}
+    {% if record.get_validate_url %}
```

Comment:
instead of using get_validate_url():

implement one new template filter:

`mrmap.main.templatetags.custom_template_filters.py`
from django.apps import apps


@register.filter
def get_model_cls(cls_name: str):
    """
    :param cls_name: django model in conventional format ==> app_label:model_name
    :type cls_name: str
    """
    app_label, model_name = cls_name.split(":")
    model_cls = apps.get_model(app_label=app_label, model_name=model_name)
    return model_cls

in template:
```
{% load custom_template_filters %}
{% get_model_cls|"quality:ConformityCheckRun.get_add_url"%}
```

I'am not 100% if the code above would work directly.

Reply:
See above. I am not sure that I get the idea completely.

Reply:
see answer above.